packages/plugins/Traderxyz/src/SNSAdaptor/SelectTokenView.tsx

```diff
@@ -0,0 +1,113 @@
+import { useI18N } from '../locales/i18n_generated'
+import { InputTokenPanel } from '../../../../../packages/mask/src/plugins/Trader/SNSAdaptor/trader/InputTokenPanel'
+import { TokenPanelType } from '../../../../../packages/mask/src/plugins/Trader/types'
```

Comment:
Please do not depends on other plugins. cc @guanbinrui maybe we should move those two things out?

Reply:
Any update on this?

Reply:
cc @guanbinrui